packages/falso/.gitignore

```diff
@@ -0,0 +1,130 @@
+# Logs
```

Comment:
Generated with
```shell
npx gitignore Node
```

Reply:
why we need this? we have one in the root.

Reply:
One in the root ignores only `node_modules` from root.
I can change `.gitignore` from root to ignore all `node_modules`, should I?